bundles/http_cache.rst

```diff
@@ -1,104 +1,86 @@
 HttpCacheBundle
 ===============
 
-The SuluHttpCache bundle provides tight integration between Sulu and HTTP caching proxies.
+The SuluHttpCache bundle provides integration between Sulu <--> FOSHttpCacheBundle <--> HTTP caching proxies.
```

Comment:
`The SuluHttpCache bundle provides integration between Sulu and HTTP caching proxies using the FOSHttpCacheBundle`

Also include a link to the `FOSHTTPCacheBundle`.